### DISCUSSION TITLE
Substring matcher

### DISCUSSION THREAD

**Discussion by dubik** [pactumjs/pactum#190] [General]
How to check that property of a json, contains a string?
```
{
  title: "Hello World"
}
```

When I use something like:
```
expectJsonLike("title", includes("World"));
```
I get an error `Json doesn't have type 'object' at '$' but found 'string'`.  I tried quite a few other combinations, but they results in a similar error.
I saw a discussion about "expectJsonSnapshot", but it complains about snapshot name, so I think it's the wrong function for my use case.
Thanks.

**Comment by ASaiAnudeep** [comment]
`expectJsonLike` doesn't support matchers. We need to use `expectJsonMatch`.

```js
const { spec } = require('pactum');
const { includes } = require('pactum-matchers');

await spec()
  .get('/some/api')
  .expectJsonMatch("title", includes("World"));
```

**Reply by dubik** [reply]
Brilliant, thank you!
